docs/connector/08-01-establish-secure-connection-with-compass.md

```diff
@@ -0,0 +1,85 @@
+# Establish a secure connection with Compass
+
+To establish a secure connection with Compass and generate the client certificate, follow this tutorial. 
+
+## Prerequisites
+
+- [OpenSSL toolkit](https://www.openssl.org/docs/man1.0.2/apps/openssl.html) to create a Certificate Signing Request (CSR), keys, and certificates which meet high security standards
+- Compass (version 1.8 or higher)
+- Registered Application
+- Runtime connected to Compass
+
+## Steps
+
+1. Get the Connector URL and the one-time token.
+
+    To get the Connector URL and the one-time token which allow you to fetch the required configuration details, use the Compass Console.
+    
+    Alternatively, make a call to the Director including the `Tenant` header with Tenant ID and `authorization` header with the Dex Bearer token. Use the following mutation: 
```

Comment:
We should provide a description of how to get Dex token. In the current state, it is not the most pleasant way to fetch it but nonetheless - there should be a guide for that.